util-liquibase-installer/pom.xml

```diff
@@ -56,7 +54,6 @@
         <dependency>
```

Comment:
There is a version listed here on line 51, this should be moved to the parent pom.

Reply:
@dermot-hardy , Is there any specific purpose for overriding postgresql version  in 
https://github.com/CAFapi/caf-common/blob/SCMOD-5073-r2.0.0/util-liquibase-installer/pom.xml#L51?
Please confirm.

Reply:
I don't think so.  It should probably just be moved to the parent POM.